tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -433,3 +434,47 @@ def test_rjumps_jumpf_nonreturning(
         possible_exceptions.append(EOFException.INVALID_NON_RETURNING_FLAG)
 
     eof_test(container=Container(sections=sections), expect_exception=possible_exceptions or None)
+
+
+def gen_stack_underflow_params():
+    """Generate parameters for stack underflow tests."""
+    for op in sorted(valid_eof_opcodes):
+        if op.popped_stack_items == 0:
+            continue
+        if op in (Op.EOFCREATE, Op.RETURNCODE):
+            continue
+        yield op, False
+        if op.popped_stack_items > 1:
+            yield op, True
```

Comment:
I think we could replace `max_height` with `stack_height` and return an integer here:
```suggestion
        yield op, 0
        if op.min_stack_height > 1:
            yield op, op.min_stack_height - 1
```
I feel it's a bit easier to understand what's happening in the test.